multitool/private/cwd.template.bat

```diff
@@ -0,0 +1,2 @@
+set tool=%cd%\{{tool}}
+cd %BUILD_WORKING_DIRECTORY% && %tool% %*
```

Comment:
is there a Windows-equivalent of `exec`?

Reply:
No, this is essentially an exec